app/migrations/versions/ac8716ecdc21_.py

```diff
@@ -0,0 +1,28 @@
+"""empty message
+
+Revision ID: ac8716ecdc21
+Revises: e2af63b3a35e
+Create Date: 2019-11-01 12:11:24.258107
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'ac8716ecdc21'
+down_revision = 'e2af63b3a35e'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_unique_constraint(None, 'user', ['_phone'])
```

Comment:
making `phone` unique per user. will this cause any issues? @enjeyw @tristanhcole

Reply:
nope this seems reasonable to me!